packages/cli/src/options/beaconNodeOptions/chain.ts

```diff
@@ -210,6 +212,16 @@ Will double processing times. Use only for debugging purposes.",
     group: "chain",
   },
 
+  "chain.archiveMode": {
+    hidden: false,
```

Comment:
this flag should probably be hidden

Reply:
It's a user facing flag not for our testing only. Also it have a default value so exposing would not harm.

Reply:
exposing it in this PR does not make sense imo, we can always do that later